test/Microsoft.ML.Benchmarks/KMeansAndLogisticRegressionBench.cs

```diff
@@ -35,7 +35,7 @@ public class KMeansAndLogisticRegressionBench
             }, hasHeader: true);
 
             var estimatorPipeline = ml.Transforms.Categorical.OneHotEncoding("CatFeatures")
-                .Append(ml.Transforms.Normalize("NumFeatures"))
+                .Append(ml.Transforms.NormalizeMinMax("NumFeatures"))
```

Comment:
>NormalizeMinMax [](start = 38, length = 15)

Great, thanks @codemzs. Only real problem I see with this API is that it may not be apparent which normalization method is the best first choice as opposed to having it be named min-max, but we can always add alias and documentation later.